config.dev.toml

```diff
@@ -144,6 +144,14 @@ TraceExporterURL = "http://localhost:14268"
 # Env overide: ATHENS_STATS_EXPORTER
 StatsExporter = "prometheus"
 
+# EtcdEndoints are comma separated URLs that determine whether to use Etcd as a distributed lock mechanism 
+# for single-flight requests. If empty, in-memory single-flight is used. If you 
+# have a single instance of Athens, then keep this empty. Otherwise, use etcd. 
+# Athens uses a single-flight mechanism to store modules into a storage Backend 
+# so that no two concurrent requests can override each other.
+# Env override: ATHENS_ETCD_ENDPOINTS
+EtcdEndpoints = ""
```

Comment:
probably related to previous comments about multiple distributed lock impl. 
we should aim for switchable implementation and configuration should reflect that. 
maybe some type at the root and vendor specific config in [EtcdSync] or something similar.
i would just move `EtcdEndpoints` under Etcd config for now and be bothered with multi vendors later